README.md

```diff
@@ -1,5 +1,9 @@
 # Roles On Routes
 
-Currently a work in progress and is unlikely to be included without error
+Roles On Routes was written by Navigating Cancer.
 
-Eventually allows defining user roles for authorization directly on the routes to which they apply
+It allows defining user roles for authorization directly on the routes to which they apply.
+
+It may not be the best solution. Maybe something open source would be better.
```

Comment:
Remove this, it probably isn't the best but at this point we're pretty married to it.

Reply:
I wanted to discourage its use in future projects, but I guess that's probably unlikely anyway.